server/cmwell-ws/app/controllers/InputHandler.scala

```diff
@@ -323,6 +327,8 @@ class InputHandler @Inject() (ingestPushback: IngestPushback,
               Future(Ok(Json.obj("success" -> true, "dry-run" -> true)))
             else {
 
+              val isPriorityWrite = req.getQueryString("priority").isDefined
```

Comment:
Makes sense. But then again, I'd have to pass it to handlePostRDF and handlePostWrapped as another Boolean. We already passing the request instance, and `.getQueryString` is O(1) since Play uses a `Map[String,Seq[String]]` for queryString. So there's no need to memoize. And using a lazy val here will actually hurt performance!
I know it feels like I'm not being DRY, but it feels more readable this way.
Let me know what do you think.